model/task/task_test.go

```diff
@@ -2035,6 +2035,7 @@ func TestActivateTasks(t *testing.T) {
 		{Id: "t2", DependsOn: []Dependency{{TaskId: "t0"}, {TaskId: "t1"}}, Activated: false, DeactivatedForDependency: true},
 		{Id: "t3", DependsOn: []Dependency{{TaskId: "t0"}}, Activated: false, DeactivatedForDependency: true},
 		{Id: "t4", DependsOn: []Dependency{{TaskId: "t0"}, {TaskId: "t3"}}, Activated: false, DeactivatedForDependency: true},
+		{Id: "t5", Activated: true},
```

Comment:
Is this supposed to depend on `t0`, so we can ascertain that `ActivateTasks` did not try to change its activation state? Otherwise, it seems like the test to `ActivateTasks` would have no effect on `t5`, even before this change.

Reply:
Also, is it worth testing direct task activation with the task ID vs. indirect activation due to dependencies?

Reply:
fixed and donw